2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_setup.py:_flush():71] setting env: {}
2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_setup.py:_flush():71] setting login settings: {}
2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_init.py:_log_setup():371] Logging user logs to /home/ck/shivani_research/TD3-error-control/wandb/run-20230615_201253-pgieqiu2/logs/debug.log
2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_init.py:_log_setup():372] Logging internal logs to /home/ck/shivani_research/TD3-error-control/wandb/run-20230615_201253-pgieqiu2/logs/debug-internal.log
2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_init.py:init():404] calling init triggers
2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'num_episodes': 10000, 'learn_every': 50, 'noise_variance': 0.1, 'test_every': 500}
2023-06-15 20:12:53,719 INFO    MainThread:593483 [wandb_init.py:init():460] starting backend
2023-06-15 20:12:53,719 INFO    MainThread:593483 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-06-15 20:12:53,725 INFO    MainThread:593483 [backend.py:ensure_launched():216] starting backend process...
2023-06-15 20:12:53,727 INFO    MainThread:593483 [backend.py:ensure_launched():221] started backend process with pid: 593532
2023-06-15 20:12:53,728 INFO    MainThread:593483 [wandb_init.py:init():469] backend started and connected
2023-06-15 20:12:53,729 INFO    MainThread:593483 [wandb_init.py:init():533] updated telemetry
2023-06-15 20:12:53,738 INFO    MainThread:593483 [wandb_init.py:init():563] communicating current version
2023-06-15 20:12:54,284 INFO    MainThread:593483 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.15.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-06-15 20:12:54,284 INFO    MainThread:593483 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2023-06-15 20:12:54,648 INFO    MainThread:593483 [wandb_init.py:init():606] starting run threads in backend
2023-06-15 20:12:56,843 INFO    MainThread:593483 [wandb_run.py:_console_start():1810] atexit reg
2023-06-15 20:12:56,843 INFO    MainThread:593483 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2023-06-15 20:12:56,843 INFO    MainThread:593483 [wandb_run.py:_redirect():1689] Redirecting console.
2023-06-15 20:12:56,844 INFO    MainThread:593483 [wandb_run.py:_redirect():1745] Redirects installed.
2023-06-15 20:12:56,844 INFO    MainThread:593483 [wandb_init.py:init():633] run started, returning control to user process
2023-06-15 20:12:56,893 INFO    MainThread:593483 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2023-06-15 20:12:56,894 INFO    MainThread:593483 [wandb_run.py:_restore():1752] restore
2023-06-15 20:12:59,353 INFO    MainThread:593483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 711
  total_bytes: 711
}

2023-06-15 20:12:59,750 INFO    MainThread:593483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 711
  total_bytes: 711
}

2023-06-15 20:13:00,665 INFO    MainThread:593483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 711
  total_bytes: 10939
}

2023-06-15 20:13:00,767 INFO    MainThread:593483 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 711
  total_bytes: 10939
}

